### DISCUSSION TITLE
Ansible Versions Required For Different OS

### DISCUSSION THREAD

**Discussion by internal-test** [CrowdStrike/ansible_collection_falcon#267] [General]
Would like to inquire about the compatible ansible versions on the following OS:

- Ubuntu 18.04
- Ubuntu 20.04
- Ubuntu 22.04
- Ubuntu 22.10
- Mac Big Sur
- Mac Monterey
- Mac Ventura

**Comment by carlosmmatos** [comment]
We currently test/support 2.11+

see https://github.com/CrowdStrike/ansible_collection_falcon#ansible-version-compatibility

That's not to say there are other possible combinations of supported versions of Ansible + OS that are currently untested and may work for you, but we chose to use 2.11+ as a good base to start from.

**Reply by internal-test** [reply]
Could you please guide me on where should i enter my API client ID and secret key?

**Reply by carlosmmatos** [reply]
I would start here: https://github.com/CrowdStrike/ansible_collection_falcon/blob/main/roles/falcon_install/README.md#falcon-api-permissions

This gives you the API scopes needed, as well as example playbooks of how you would pass in the client id and secret.
